SampleApp/BoxView.swift

```diff
@@ -9,31 +9,155 @@ import SwiftUI
 import Warp
 
 struct BoxView: View {
+    @State private var style: Warp.BoxStyle = .info
+
+    @State private var shouldShowIcon = true
+
+    @State private var boxTitle = "Title"
+
+    @State private var boxSubtitle = "Use this variant to call extra attention to useful, contextual information."
+
+    @State private var hasLink = false
+
+    @State private var hasButton = false
+
+    @Environment(\.colorScheme) private var colorScheme
+
     var body: some View {
-        ScrollView(showsIndicators: false) {
-            ForEach(Warp.BoxStyle.allCases, id: \.self) { style in
-                Warp.Box(
```

Comment:
Our components should by default have a padding.
![Simulator Screenshot - iPhone 15 - 2024-02-13 at 20 09 05](https://github.com/warp-ds/warp-ios/assets/89784012/c4fa4b35-3f44-4c81-a6a3-1d310a3bfd98)

Reply:
Thanks for the finding, you mean should I add paddings to the box itself?

Reply:
yes, inside the component we should add padding that users should use, so different users do not add random padding for their cases.

Reply:
Don't worry about it thou if it requires too much changes. We should sync this across all our components probably. The things that should be full screen need to have a padding that we can control and not the user. We have not started working on the padding, our paddings are static now which should not be the case and should be dynamic and a multiple of 4 probably.